src/main/java/com/byclosure/maven/plugins/middleman/AbstractJRubyMojo.java

```diff
@@ -136,37 +139,36 @@ protected CommandLine getCrossPlatformCommandLine(String executable) {
 		} else {
 			cmdLine = new CommandLine(executable);
 		}
-
 		return cmdLine;
 	}
 
-	protected void executeCommandLine(CommandLine cmdLine, Map<String, String> env, File workingDir) throws IOException {
-		DefaultExecutor executor = new DefaultExecutor();
+	protected void executeCommandLine(CommandLine cmdLine,
+			Map<String, String> env, File workingDir) throws IOException {
+		final DefaultExecutor executor = new DefaultExecutor();
 		executor.setWorkingDirectory(workingDir);
-
-		PumpStreamHandler pump = new PumpStreamHandler(System.out, System.err, System.in);
+		final PumpStreamHandler pump = new PumpStreamHandler(System.out,
+				System.err, System.in);
 		executor.setStreamHandler(pump);
-
 		executor.execute(cmdLine, env);
 	}
 
 	protected Map<String, String> getEnv() {
-		final Map<String, String> env = new HashMap<String, String>(System.getenv());
+		final Map<String, String> env = new HashMap<String, String>(
+				System.getenv());
+		for (String propertyName : System.getProperties().stringPropertyNames()) {
+			env.put(propertyName, System.getProperty(propertyName));
+		}
```

Comment:
We could pass as Environment variables or System Properties (with -D or -J-D). But, because we are launching a new process with bundler the -J-D properties would be removed, so we had to pass them as Environment variables.